bin/run_server.jl

```diff
@@ -27,6 +27,8 @@ timestamp_logger(logger) = TransformerLogger(logger) do log
     merge(log, (; message = "[$(Dates.format(now(), date_format))] $(log.message)"))
 end
 
+compress_logs = get(ENV, "JULIA_PKG_SERVER_COMPRESS_LOGS", "true") in ("True", "TRUE", "true")
```

Comment:
Should probably be const or at least typed.

Reply:
I supposed we should do that across the file as well